onnxscript/test/analysis_test.py

```diff
@@ -96,8 +97,8 @@ def while_eg(x):
 
 class TestExposedUses(unittest.TestCase):
     def assertUses(self, f, expected):
-        ast = main.get_ast(f)  # pylint: disable=redefined-outer-name
-        result = analysis.exposed_uses(ast.body, converter.Converter())
+        source, ast = main.get_src_and_ast(f)  # pylint: disable=redefined-outer-name
```

Comment:
Since this line is touched - possible to rename `ast` to avoid `disable=redefined-outer-name`?

Reply:
Done